src/golang/lib/context/context.go

```diff
@@ -21,7 +23,8 @@ const (
 
 type AqContext struct {
 	user.User
-	RequestId string
+	RequestId     string
+	StorageConfig *shared.StorageConfig
```

Comment:
Can you add a comment here that the storage config is a snapshot that is only meant to live for the length of the request. I think that would be super helpful for readers